lib/src/widgets/animated_book_widget.dart

```diff
@@ -89,6 +109,23 @@ class _AnimatedBookWidgetState extends State<AnimatedBookWidget>
   late double spreadRadius = widget.spreadRadius;
   late Offset backgroundBlurOffset = widget.backgroundBlurOffset;
 
+  @override
+  void initState() {
+    super.initState();
+    animationController = AnimationController(
```

Comment:
This can be as follows:
```dart
late AnimationController animationController = AnimationController(
      this, 
      widget.animationDuration ?? _defaultAnimationDuration,
      reverseDuration: widget.reverseAnimationDuration ?? _defaultReverseAnimationDuration,
    )..addStatusListener(statusListener);
```

There is no need to use the initState